app/controllers/authentication_tokens/revokes_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module AuthenticationTokens
+  class RevokesController < ApplicationController
+    def show
+      authorize(token)
+    end
+
+    def update
+      authorize(token)
+
+      token.revoke!
+
+      redirect_to(authentication_tokens_path)
+    end
+
+  private
+
+    def token
+      @token ||= policy_scope(AuthenticationToken).where(id: token_params).first
+    end
+
+    def token_params
+      params.require(:authentication_token_id)
+    end
```

Comment:
```suggestion
```

Reply:
I know it's only one field, but I twitch at not putting params through strong params, even for finds.

Reply:
It's not needed because if `params[:authentication_token_id]` is `nil` Rails will raise an `ActiveRecord::RecordNotFound` when we attempt to find that record and will result in a `404`.